### ISSUE TITLE
3.2.4 breaks calc with css variables

### ISSUE THREAD

**Issue by klimashkin** [JPeer264/node-rename-css-selectors#43]
Hi! Even with `ignoreCssVariables: true` the new version changes this css
```css
margin-right: calc(var(--gap-column-child, var(--0px)) - var(--gap-column, var(--0px)));
margin-bottom: calc(var(--gap-row-child, var(--0px)) - var(--gap-row, var(--0px)));
```
into
```
margin-right:calc(var(--gap-column-child, var(--0px));
margin-bottom:calc(var(--gap-row-child, var(--0px));
```
which breaks the whole css file (parens missing) : )

**Comment by JPeer264** [comment]
Oh! I'm gonna  check that asap !

**Comment by JPeer264** [comment]
Fixed in v`3.2.5`
